docker-compose.yml

```diff
@@ -26,18 +26,32 @@ services:
       POSTGRES_PASSWORD: postgres
     volumes:
       - ./data/txs-db:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
 
   cfg-db:
     image: postgres:14-alpine
     environment:
       POSTGRES_PASSWORD: postgres
     volumes:
       - ./data/cfg-db:/var/lib/postgresql/data
-
+    healthcheck:
```

Comment:
Could you inherit using `<<:` yaml so you don't have to define the same liveness multiple times for redis and postgres?